### PR TITLE
[Merged by Bors] - Adds documentation for FUSE

### PR DESCRIPTION
**Pull request by lfrancke**
# Description

Adds documentation on how to use FUSE

## Definition of Done Checklist

- Not all of these items are applicable to all PRs, the author should update this template to only leave the boxes in that are relevant
- Please make sure all these things are done and tick the boxes

```[tasklist]
# Reviewer
- [x] Code contains useful comments
- [x] Documentation added or updated
```

```[tasklist]
# Acceptance
- [ ] Proper release label has been added
```

Once the review is done, comment `bors r+` (or `bors merge`) to merge. [Further information](https://bors.tech/documentation/getting-started/#reviewing-pull-requests)

### CODE REVIEW

File: docs/modules/hdfs/pages/usage-guide/fuse.adoc [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+Inside this Pod you can get a shell (e.g. using `kubectl exec --stdin --tty hdfs-fuse -- /bin/bash`) to get access to a script called `fuse_dfs_wrapper` (it is in the `PATH` of our Hadoop images).
+
+To see the available options, call the script without any parameters.
+
+To mount HDFS call the script like this:
+
+[source,bash]
+----
+fuse_dfs_wrapper.sh dfs://<your hdfs> <target> <1> <2>
```
Comment (maltesander) [inline docs/modules/hdfs/pages/usage-guide/fuse.adoc]: The script is only called `fuse_dfs_wrapper` https://github.com/stackabletech/docker-images/blob/18d1366cebdfea4093f95f9c6681faec8e3dc812/hadoop/Dockerfile#L130C61-L130C61
(the same for the other usages)

File: docs/modules/hdfs/pages/usage-guide/fuse.adoc [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+# This will run in debug mode and stay in the foreground
+fuse_dfs_wrapper.sh -odebug dfs://<your hdfs> <target>
+
+# Example:
+mkdir simple-hdfs
+fuse_dfs_wrapper.sh dfs://simple-hdfs simple-hdfs
+cd simple -hdfs
```
Comment (maltesander) [inline docs/modules/hdfs/pages/usage-guide/fuse.adoc]:
```suggestion
cd simple-hdfs
```

**Review by maltesander — CHANGES_REQUESTED** [review]
Minor things, LGTM otherwise!

File: docs/modules/hdfs/pages/usage-guide/fuse.adoc (hunk 1 above)
Comment (lfrancke) [reply docs/modules/hdfs/pages/usage-guide/fuse.adoc]: Thank you! Good find.

**Review by maltesander — APPROVED** [review]
LGTM!

**Comment by lfrancke** [comment]
Thank you. I'll wait with merging until the other PR is done in case anything changes there.

**Comment by lfrancke** [comment]
bors merge

**Pull request closed without merging**